qmcpy/discrete_distribution/_discrete_distribution.py

```diff
@@ -52,6 +52,39 @@ def pdf(self, x):
         """ ABSTRACT METHOD to evaluate pdf of distribution the samples mimic at locations of x. """
         raise MethodImplementationError(self, 'pdf')
 
+    def plot(self, n, d_vertical=0, d_horizontial =1):
+        """
+        ABSTRACT METHOD for ploting a distrubution, you will also need to import matplotlib.pyplot as plt
```

Comment:
Although this method is in the abstract class, this is not an abstract method. 

You should `impot matplotlib.pyplot as plt` in the routine. If you put this import in a try-except then you can throw a friendly error for the user who tries to call this routine without having matplotlib installed.